docs/sources/configure-client/trace-span-profiles/_index.md

```diff
@@ -21,11 +21,12 @@ Key benefits and features:
 Get started:
 
 - Configure Pyroscope: Begin sending profiling data to unlock the full potential of Span Profiles
-- Client-Side Packages: Easily link traces and profiles using available packages for Go, Java, Ruby and .NET
+- Client-Side Packages: Easily link traces and profiles using available packages for Go, Java, Ruby, .NET and Python
```

Comment:
We use sentence case in our documentation. Not everything is consistent yet, but we're working on it.
```suggestion
- Client-Side packages: Easily link traces and profiles using available packages for Go, Java, Ruby, .NET and Python
```